recipes/twilio/meta.yaml

```diff
@@ -0,0 +1,213 @@
+{% set name = "twilio" %}
+{% set version = "6.32.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e78a2006b9449fb9fad5050537e0998c181c7d3a62eaa9eed434e59dbaf58324
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
```

Comment:
in the ``host`` section I think you just need ``pip``, ``python`` and ``setuptools``

Reply:
Thanks, only `pip` and `python` seem enough.